MaterialComponentsBeta.podspec

```diff
@@ -104,6 +104,23 @@ Pod::Spec.new do |mdc|
     end
   end
 
+  mdc.subspec "Tabs+TabBarView" do |extension|
```

Comment:
_Nit:_ Shouldn't this be the component name? `TabBarView` instead of `Tabs+TabBarView`?

Reply:
This is built as an extension on top of Tabs. "TabBarView" implies it's a brand new component and not related to Tabs.  Is that what you expect this to be?

Reply:
Switched to "Tabs2" since the component isn't only a view.